source/core/parameters_lagrangian.cc

```diff
@@ -64,6 +72,23 @@ namespace Parameters
       particle_average_diameter.at(particle_type) = prm.get_double("diameter");
       particle_size_std.at(particle_type) =
         prm.get_double("standard deviation");
+      particle_histogram_diameter.at(particle_type) =
+        entry_string_to_vector(prm, "histogram diameters list");
+      particle_histogram_probability.at(particle_type) =
+        entry_string_to_vector(prm, "histogram probabilities list");
+
+      double probability_sum =
+        std::reduce(particle_histogram_probability.at(particle_type).begin(),
+                    particle_histogram_probability.at(particle_type).end());
+
+      // We make sure that the cummulative probability is equal to 1.
+      if (probability_sum != 1.0)
```

Comment:
I'm scared that it could be true even if the sum=1.0 because of the machine accuracy
```suggestion
      if (std::abs(probability_sum-1.0) > 1e-12)
```

Reply:
Even so, I think it is a good pratrice to avoid double comparison :)